src/client/components/App.tsx

```
@@ -374,6 +381,28 @@ export default function App() {
         };
         break;
 
+      case "conversation.item.input_audio_transcription.completed":
+      case "conversation.item.input_audio_transcription.failed":
+        if (event.error) {
+          console.error("error: conversation.item.input_audio_transcription.failed:", event.error);
+        }
+
+        const msgId = event.item_id;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)